openlane/steps/odb.py

```diff
@@ -718,6 +718,10 @@ def run(self, state_in: State, **kwargs) -> Tuple[ViewsUpdate, MetricsUpdate]:
             info("'DIODE_ON_PORTS' is set to 'none': skipping…")
             return {}, {}
 
+        if self.config["DIODE_CELL"] is None:
+            info("DIODE_CELL not set. Skipping…")
```

Comment:
add `{self.id}` for consistency with https://github.com/efabless/openlane2/blob/53f5e7d4b7c62c7eaee1d2789ea8744210519c11/openlane/steps/openroad.py#L1162

Reply:
Agreed